autogen.sh

```diff
@@ -74,7 +75,7 @@ rm -rf Makefile \
        configure
 
 if [[ $lark_shovel ]]; then
-    rm -Rfv lib/lark lib/lark-stubs lib/lark*.dist-info
+    rm -Rfv lib/lark lib/lark-stubs lib/lark*.dist-info lib/lark*.egg-info
 fi
```

Comment:
Question: why do we avoid removing it by default? What makes the inverse, e.g., remove by default unless specified otherwise more appealing?

Reply:
It has to download the packages from the internet, which I didn't want to do every time `autogen.sh` is run.